models/marts/dim_customers.sql

```diff
@@ -1,5 +1,34 @@
 with customers as  (
-    select * from {{ ref('stg_customers' )}}
+
+    select * from {{ ref('stg_customers')}}
+
+),
+
+orders as  (
+
+    select * from {{ ref('stg_orders')}}
+
+),
+
+customer_first_last_order_at as (
+
+    select
+        customer_id,
+        first_value(orders.ordered_at) over(partition by orders.customer_id order by orders.ordered_at asc) as first_order_at,
+        last_value(orders.ordered_at) over(partition by orders.customer_id order by orders.ordered_at asc) as last_order_at
```

Comment:
With SQLFluff we can enforce line length limits. Here is an example where it might be beneficial to break up these columns into multi-line statements. No need to change but just for you to think about when writing SQL

Reply:
Modified